src/plugins/pcr/src/InSilicoPcrTask.cpp

```diff
@@ -111,6 +111,13 @@ void InSilicoPcrTask::prepare() {
     CHECK_OP(stateInfo, );
     FindAlgorithmTaskSettings reverseSettings = getFindPatternSettings(U2Strand::Complementary);
     CHECK_OP(stateInfo, );
+    if (qgetenv(ENV_GUI_TEST) == "1") {
```

Comment:
This way you may affect other tests
Could you do the following
1) Use some bigger max values - not 1 but a more reasonable 10 for example?
2) Use a dedicated variable: UGENE_MAX_RESULTS_FOR_PRIMERS_PER_STRAND. Set it in the first line of you GUI test.